frontend/packages/dev-console/integration-tests/features/topology/topology-smoke_test.feature

```diff
@@ -0,0 +1,165 @@
+Feature: Topology smoke test
+	As a user, I want to verify topology feature    
```

Comment:
Can you please update it to something like
```suggestion
	User should be able to use topology feature
```
Because user won't verify the topology feature, he will use them.

Reply:
It will be valid for every feature we write for eg:use pipeline feature or use build feature . I think better would be to verify basic topology features